chain/interface.go

```diff
@@ -10,6 +10,16 @@ import (
 	"github.com/roasbeef/btcwallet/wtxmgr"
 )
 
+// BackEnds returns a list of the available back ends.
+// TODO: Refactor each into a driver and use dynamic registration.
+func BackEnds() []string {
+	return []string{
+		//"btcd",
```

Comment:
Seems the rest of these shouldn't _still_ be commented out?

Reply:
I uncommented them; this was so that I could run tests faster while working on the bitcoind implementation.